platform/android/export/export_plugin.cpp

```diff
@@ -223,6 +223,18 @@ static const char *MISMATCHED_VERSIONS_MESSAGE = "Android build version mismatch
 
 static const char *GDEXTENSION_LIBS_PATH = "libs/gdextensionlibs.json";
 
+// This template string must always match the content of icon.xml file.
+static const String ICON_XML_TEMPLATE =
+		"<?xml version=\"1.0\" encoding=\"utf-8\"?>\n"
+		"<adaptive-icon xmlns:android=\"http://schemas.android.com/apk/res/android\">\n"
+		"    <background android:drawable=\"@mipmap/icon_background\"/>\n"
+		"    <foreground android:drawable=\"@mipmap/icon_foreground\"/>\n"
+		"%s" // Placeholder for the optional monochrome tag.
+		"</adaptive-icon>";
+
+static const String icon_xml_path = "res/mipmap-anydpi-v26/icon.xml";
+static const String themed_icon_xml_path = "res/mipmap-anydpi-v26/themed_icon.xml";
```

Comment:
For style consistency, let's use upper case for those constants.. and yes, I'm aware the constants below do not follow this style.. another cleanup I need to do eventually :(